Tanasee/Tanasee1.xml

```diff
@@ -53,6 +135,1087 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                            ሊቀ፡ ማእምራን፡ <persName ref="PRS14067HabtuSarad">ኀብቱ፡ ጸራድ፡</persName> ማስሬ፡ አቀበስን፡ 
+                                            ቀይስ፡ ሐፄ፡ <persName ref="PRS14068TaklaMaryam">ተክለ፡ ማርያም፡</persName> 
+                                            ሊቀ፡ ደብተራ፡ <persName ref="PRS14069TaklaNabiyat">ተከለ፡ ነቢያት፡</persName> 
+                                            ቀኚዕ፡ ብቶደደ፡ <persName ref="PRS14071HabtaMG">ሐብተ፡ ማርያም፡</persName> 
+                                            ግራ፡ ብቶደደ፡ <persName ref="PRS14072Naze">ናዜ፡</persName> 
+                                            አዛዘ፡ እንዳዘዜ፡ ርእሰ፡ ርኡሳን፡ <persName ref="PRS14073TaklaMaryam">ተከለ፡ ማርያም፡</persName> 
+                                            እንዘ፡ ንቡረ፡ ዕድ፡ <persName ref="PRS14074Gersam">ጌርሣም፡</persName> 
+                                            ወምስለ፡ ንብርዕድ፡ <persName ref="PRS14075Tiwogolos">ቲዎጎሎስ፡</persName> 
+                                            ወቂስ፡ ገበዝ፡ <persName ref="PRS14076GabraNazrawi">ገብረ፡ ናዝራዊ፡</persName> 
+                                            ወሊቀ፡ ዲየቆን፡ ይፅናዩ፡ ወራቍ፡ ማስራ፡ <persName ref="PRS14077FereKrestos">ፍሬ፡ ክርስቶስ፡</persName> ዘተመይጠ፡ ንዋይ፡ ለታቦተ፡ 
+                                            ገብርኤል፡ ዘደብረ፡ ክብራ፡ ፳ወ፫፡ ንዋየ፡ ቤተ፡ ክርስቲያን፡</q>
+                                </item>
+                                <item xml:id="a11">
+                                    <locus target="#4va"/>
+                                     <desc type="Inventory">Note on church utensils by a rather early hand written on a previously effaced part of the folio.</desc>
+                                    <q xml:lang="gez">ይቤሉ፡ ቅዱሳን፡ ዘደብረ፡ ክብራ፡ በእንተ፡ ሠርፀ፡ ማርያም፡ ቀሲስ፡ ዘጻመወ፡ ላቲ፡ ለቤተ፡ ክርስቲያን፡ ፴ወ፪፡ ወቂት፡ ወርቅ። አልህምት፡ ፴ወ፫። መስቀል፡ ፫፡ ታቦታት፡ ፯። 
+                                        <gap reason="ellipsis" unit="lines" quantity="7"/> ምጽሕርት፡ ዘኀፂን። </q>
+                                </item>
+                                <item xml:id="a12">
+                                    <locus target="#4vb"/>
+                                    <desc type="Inventory">Possibly the last line of an inventory, that was written on the first column of this folio, effaced and rewritten 
+                                        with another text in <ref target="#a13"/>.</desc>
+                                    <q xml:lang="gez">ዘኀፂ<gap reason="illegible" unit="chars" quantity="2"/>ላ። አርጋኖን። ፪ሥዕል። </q>
+                                </item>
+                                <item xml:id="a13">
+                                    <locus target="#4vb"/>
+                                    <desc type="MixedNote">Commemorative note (Tazkār) on a person named 
+                                        <persName ref="PRS14080SardaMaryam">ሠርፀ፡ ማርያም፡</persName>, who donated to the church of Māryām by
+                                        a similar hand as <ref target="#a14 #a15"/>.</desc>
```

Comment:
I would split the refs and add the text to be printed nicely
`similar hand as <ref target="#a14">additio 14</a> and <ref target="#a15">additio 15</a>`

Reply:
Done.